### PR TITLE
Running man + Forum mention

### LINKED ISSUE #17: Add code to change running widget position
See [thread](https://discuss.streamlit.io/t/change-the-running-widget-position/30466)

### PR DESCRIPTION
**Pull request by arnaudmiribel**
Fixes #17 
Also adds an optional mention to a forum thread whenever applicable

### CODE REVIEW

File: gallery/streamlit_app.py [hunk 1]
```diff
@@ -166,6 +169,9 @@ def page_content():
             if buymeacoffee_username:
                 with next(columns):
                     badge("buymeacoffee", name=buymeacoffee_username)
+            if forum_url:
+                with next(columns):
+                    mention(icon="streamlit", url=forum_url, label="Forum post")
```
Comment (blackary) [inline gallery/streamlit_app.py:174]: Ooh, good idea. Great to get more people looking at the forum. Will need to go and back-fill a bunch of the forum post links.

**Review by blackary — CHANGES_REQUESTED** [review]
Can you add a test for this?

File: src/streamlit_extras/customize_running/__init__.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import time
+
+import streamlit as st
+
+center_css = """
+<style>
+
+div[class='css-4z1n4l ehezqtx5']{
```
Comment (blackary) [inline src/streamlit_extras/customize_running/__init__.py:8]: So, I am always too nervous to use random-looking class names like this, as I feel like they might be liable to change at any time. Do you have any idea if these are likely to stay the same, long-term? Happy to ask internally if you don't know, but didn't know if you'd already asked.

Comment (arnaudmiribel) [reply src/streamlit_extras/customize_running/__init__.py]: Yea I wanted to give a double check. Found a cleaner class here https://github.com/arnaudmiribel/streamlit-extras/pull/36/commits/633c6477d1ff553a6fb8f923d92f6b9e0e722778#diff-038353bf37afcfd34ef32ecc32752fdc0b975e6ff65471b8299dbceda013587dR8

**Comment by arnaudmiribel** [comment]
> Can you add a test for this?

Added in [633c647](https://github.com/arnaudmiribel/streamlit-extras/pull/36/commits/633c6477d1ff553a6fb8f923d92f6b9e0e722778)

**Review by blackary — APPROVED** [review]
👍

**Pull request merged**